ztp/ztp-ran-policy-generator/ranGenTemplateExamples/group-du-sno-ranGen.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: ranPolicyGenerator/v1
+kind: RanGenTemplate
+metadata:
+  # The name will be used to generate the placementBinding and placementRule names as {name}-placementBinding and {name}-placementRule
+  name: "group-du-sno-policies"
+  namespace: "policy-template"
+  labels:
+    common: false
+    groupName: "group-sno-du"
+    siteName: "N/A"
+    # In this example we used the MCP=worker-du for group-du-sno to let the clusters that will be under group-du-sno also have the policies for group-du
+    mcp: "worker-du"
```

Comment:
I believe that SNO clusters have master and worker MCP and we cannot move the node to any other. That would change this to "master" and would require the above group-du-ranGen.yaml to also be changed to "master" for SNO.

Reply:
Is that more MCP issue ? cause I do remember when i test the du-sno the policies get applied but the MCP keep showing 0 node available

Reply:
You need to use the master mcp for SNO.

Reply:
Correct, if you do not use master the cluster gets confused and doesn't know what to do with the node. It will see that the node meets the selection criteria for master as well as whatever else you might create and gives up. The result is that the node ends up not in any MCP which causes more issues later on. Until/unless SNO supports multiple machine pools we need to put everything in master.